packages/back-end/src/app/controllers/nf-tower/workflow/create-workflow-execution.lambda.ts

```diff
@@ -95,15 +104,36 @@ export const handler: Handler = async (
     console.log('Next Flow Tower API create workflow launch API Parameters:', apiQueryParameters);
     console.log('Next Flow Tower API create workflow launch request:', createWorkflowLaunchRequest);
 
-    const response: CreateWorkflowLaunchResponse = await httpRequest<CreateWorkflowLaunchResponse>(
+    const nfTowerResponse: CreateWorkflowLaunchResponse = await httpRequest<CreateWorkflowLaunchResponse>(
       `${process.env.SEQERA_API_BASE_URL}/workflow/launch?${apiQueryParameters}`,
       REST_API_METHOD.POST,
       { Authorization: `Bearer ${accessToken}` },
       createWorkflowLaunchRequest, // Delegate request body validation to Seqera Cloud / NextFlow Tower
     );
-    console.log('Create workflow launch response from Next Flow Tower API:', response);
+    console.log('Create workflow launch response from Next Flow Tower API:', nfTowerResponse);
+
+    // Create laboratory run
+    const runId: string = crypto.randomUUID().toLowerCase();
```

Comment:
This `runId` value should be the `transactionId` that is available on the FE's Stepper UI, but it is not currently passed across to the `/nf-tower/workflow/create-workflow-execution` API call at the moment.

We can address this in a separate PR that also updates the FE to pass the `transactionId` value.